4-storage/storage-loader/lib/snowplow-storage-loader/redshift_loader.rb

```diff
@@ -103,14 +96,19 @@ def load_events_and_shredded_types(config, target, snowplow_tracking_enabled)
         # and execute them in series. VACUUMs cannot be performed
         # inside of a transaction
         if config[:include].include?('vacuum')
-          vacuum_statements = [
-            build_vacuum_statement(target[:table])
-          ]
-          vacuum_statements.push(*shredded_statements.map(&:vacuum).uniq)
-
-          status = PostgresLoader.execute_queries(target, vacuum_statements)
-          unless status == []
-            raise DatabaseLoadError, Sanitization.sanitize_message("#{status[1]} error executing VACUUM statements: #{status[0]}: #{status[2]}", credentials)
+          vacuum_statements = [build_vacuum_statement(target[:table])] + shredded_statements.map(&:vacuum).uniq
+          vacuum_status = PostgresLoader.execute_queries(target, vacuum_statements)
+          unless vacuum_status == []
+            raise DatabaseLoadError, Sanitization.sanitize_message("#{vacuum_status[1]} error executing VACUUM statements: #{vacuum_status[0]}: #{vacuum_status[2]}", credentials)
+          end
+        end
+
+        # ANALYZE statements should be executed after VACUUM statements.
+        unless config[:skip].include?('analyze')
+          analyze_statements = [build_analyze_statement(target[:table])] + shredded_statements.map(&:analyze).uniq
+          analyze_status = PostgresLoader.execute_transaction(target, analyze_statements)
+          unless analyze_status == []
+            raise DatabaseLoadError, Sanitization.sanitize_message("#{analyze_status[1]} error executing ANALYZE statements: #{analyze_status[0]}: #{analyze_status[2]}")
```

Comment:
@fblundun this call to `sanitize_message` is missing the second argument, to `credentials`